src/main/resources/test_files/runtime/TypeChecking/GroupByClauseType2.jq

```diff
@@ -0,0 +1,4 @@
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:3:COLUMN:25:" :)
+for $x in ("foo", "bar")
+group by $y as string := string-length($x)
+return $y
```

Comment:
This file should be named as 'GroupByClauseTypeCheckError[number].jq'

Reply:
DONE